.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This configuration comes from https://github.com/Microsoft/vscode-recipes/tree/master/nodemon#configure-vs-code-debugging-with-a-launchjson-file